File: OneCodeTeam/How-to-use-Bing-Translator-API-in-Windows-Azure-25733/[C#]-How to use Bing Translator API in Windows Azure/C#/CSAzureBingTranslator/WebRole1/App_Code/AdmAccessToken.cs
/****************************** Module Header ******************************\
Module Name:  AdmAccessToken.cs
Project:      CSTranslatorForAzure
Copyright (c) Microsoft Corporation.
 
The sample code demonstrates how to use Bing translator API when you get it 
from Azure marked place.

This is a Admin Access Token entity class.
 
This source is subject to the Microsoft Public License.
See http://www.microsoft.com/en-us/openness/resources/licenses.aspx#MPL
All other rights reserved.
 
THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\***************************************************************************/


using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

using System.Runtime.Serialization;


namespace TranslatorForAzure
{
    /// <summary>
    /// 
    /// </summary>
    [DataContract]
    public class AdmAccessToken
    {
        [DataMember]
        public string access_token { get; set; }
        [DataMember]
        public string token_type { get; set; }
        [DataMember]
        public string expires_in { get; set; }
        [DataMember]
        public string scope { get; set; }
    }
}

File: OneCodeTeam/How-to-use-Bing-Translator-API-in-Azure-VBNET-20229/[VB]-How to use Bing Translator API in Azure [VB.NET-Visual Studio 2010]/VB/VBAzureBingTranslator/WebRole1/App_Code/AdmAccessToken.vb
'***************************** Module Header ******************************\
'Module Name:  AdmAccessToken.vb
'Project:      VBTranslatorForAzure
'Copyright (c) Microsoft Corporation.
' 
'The sample code demonstrates how to use Bing translator API when you get it 
'from Azure marked place.
'
'This is a Admin Access Token entity class.
' 
'This source is subject to the Microsoft Public License.
'See http://www.microsoft.com/en-us/openness/resources/licenses.aspx#MPL
'All other rights reserved.
' 
'THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
'EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
'WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'\**************************************************************************



Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Runtime.Serialization


Namespace TranslatorForAzure
    <DataContract()>
    Public Class AdmAccessToken
        Private _access_token As String
        Private _token_type As String
        Private _expires_in As String
        Private _scope As String

        <DataMember()>
        Public Property access_token() As String
            Get
                Return _access_token
            End Get
            Set(value As String)
                _access_token = value
            End Set
        End Property

        <DataMember()>
        Public Property token_type() As String
            Get
                Return _token_type
            End Get
            Set(value As String)
                _token_type = value
            End Set
        End Property

        <DataMember()>
        Public Property expires_in() As String
            Get
                Return _expires_in
            End Get
            Set(value As String)
                _expires_in = value
            End Set
        End Property

        <DataMember()>
        Public Property scope() As String
            Get
                Return _scope
            End Get
            Set(value As String)
                _scope = value
            End Set
        End Property

    End Class
End Namespace